src/components/common/BackNavigation.tsx

```diff
@@ -0,0 +1,20 @@
+import { BsArrowLeftShort } from 'react-icons/bs';
+import { Link } from 'react-router-dom';
+
+type BackNavigation = {
+  route: string;
+  size: number;
+};
```

Comment:
```suggestion
type BackNavigationProps = {
  route: string;
  size: number;
};
```

props와 관련된 타입 별칭이라서 마지막에 props 넣어주세요.

Reply:
넵 BackNavigationProps로 수정했습니다!